tests/vm_watch_test.go

```diff
@@ -213,6 +214,77 @@ var _ = Describe("[rfe_id:3423][crit:high][vendor:cnv-qe@redhat.com][level:compo
 		Expect(err).ToNot(HaveOccurred())
 		Expect(titles).To(Equal([]string{"NAME", "AGE", "STATUS", "VOLUME"}),
 			"Output should have the proper columns")
+
+		vmStatus, err := readNewStatus(stdout, titles, readTimeout)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmStatus).To(ConsistOf(vm.Name, MatchRegexp(vmAgeRegex), string(v12.VirtualMachineStatusStopped)),
+			"VM should be in the %s status", v12.VirtualMachineStatusStopped)
+
+		By("Starting the VM")
+		vm = tests.StartVirtualMachine(vm)
+
+		vmStatus, err = readNewStatus(stdout, vmStatus, readTimeout)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmStatus).To(ConsistOf(vm.Name, MatchRegexp(vmAgeRegex), string(v12.VirtualMachineStatusStarting)),
+			"VM should be in the %s status", v12.VirtualMachineStatusStarting)
+
+		vmStatus, err = readNewStatus(stdout, vmStatus, readTimeout)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmStatus).To(ConsistOf(vm.Name, MatchRegexp(vmAgeRegex), string(v12.VirtualMachineStatusRunning)),
+			"VM should be in the %s status", v12.VirtualMachineStatusRunning)
+
+		By("Pausing the VirtualMachine")
+		pauseCommand := tests.NewRepeatableVirtctlCommand(virtctlpause.COMMAND_PAUSE, "vm", "--namespace", vm.Namespace, vm.Name)
+		Expect(pauseCommand()).To(Succeed())
+
+		vmStatus, err = readNewStatus(stdout, vmStatus, readTimeout)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmStatus).To(ConsistOf(vm.Name, MatchRegexp(vmAgeRegex), string(v12.VirtualMachineStatusPaused)),
+			"VM should be in the %s status", v12.VirtualMachineStatusPaused)
+
+		By("Unpausing the VirtualMachine")
+		unpauseCommand := tests.NewRepeatableVirtctlCommand(virtctlpause.COMMAND_UNPAUSE, "vm", "--namespace", vm.Namespace, vm.Name)
+		Expect(unpauseCommand()).To(Succeed())
+
+		vmStatus, err = readNewStatus(stdout, vmStatus, readTimeout)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmStatus).To(ConsistOf(vm.Name, MatchRegexp(vmAgeRegex), string(v12.VirtualMachineStatusRunning)),
+			"VM should be in the %s status", v12.VirtualMachineStatusRunning)
+
+		By("Migrating the VirtualMachine")
+
+		// Verify we have more than one scheduleable node
+		virtClient, err := kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+
+		nodes := tests.GetAllSchedulableNodes(virtClient)
+		if len(nodes.Items) > 1 {
```

Comment:
@zcahana I think that for the sake of coverage, we either should add another test case where we strictly demand more than 1 scheduable node in order to test the migration output, or we should demand it here in this test.

Reply:
Changed so that the current testcase requires #nodes >= 2.